src/stylesheets/header.scss

```diff
@@ -6,7 +6,7 @@ header.reusable-header {
   > span {
     display: block;
     float: left;
-    line-height: 2;
+    line-height: 1.5;
```

Comment:
Can you vertically center the log out button?  It's misaligned with the header text now.